packages/react-instantsearch-dom/src/server.js

```diff
@@ -1,3 +1 @@
-import createInstantSearchServer from './core/createInstantSearchServer';
-
-export const createInstantSearch = createInstantSearchServer;
+export { findResultsState } from './core/createInstantSearchServer';
```

Comment:
Is prefixing the method `findResultsState` with `find` a convention? I've always found it weird in this context.

Reply:
Yes it's a convention for our SSR API. It's maybe a bit counter-intuitive but we have the same inside Vue InstantSearch. Until we have a dedicated API inside InstantSearch we should stay with this one IMO.